tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -130,6 +130,9 @@ public override bool Execute ()
             File.Copy(Path.Join (MicrosoftNetCoreAppRuntimePackDir, "native", f), Path.Join(AppDir, f), true);
         File.Copy(MainJS!, Path.Join(AppDir, "runtime.js"),  true);
 
+        var html = @"<html><body><script type=""text/javascript"" src=""runtime.js""></script></body></html>";
+        File.WriteAllText (Path.Join (AppDir, "index.html"), html);
```

Comment:
```suggestion
        File.WriteAllText(Path.Join(AppDir, "index.html"), html);
```

In dotnet/runtime we don't use spaces in between `(` and the method identifier in managed code.